### ISSUE TITLE
[BUG] error syncing ntp-servers

### ISSUE THREAD

**Issue by brandboat** [harvester/harvester#6249]
**Describe the bug**
After build cluster from the master branch, in harvester pod I saw bunch of error logs
```text
time="2024-08-01T06:43:51Z" level=info msg="Processing setting ntp-servers, value: {\"ntpServers\":[\"0.suse.pool.ntp.org\",\"1.suse.pool.ntp.org\"]}"
time="2024-08-01T06:43:51Z" level=error msg="Create node config with nodeName harvester-node-0 failure: the object provided is unrecognized (must be of type NodeConfig): Object 'Kind' is missing in '{\"metadata\":{\"name\":\"harvester-node-0\",\"namespace\":\"harvester-system\",\"creationTimestamp\":null},\"spec\":{\"ntpConfigs\":{\"ntpServers\":\"0.suse.pool.ntp.org 1.suse.pool.ntp.org\"}},\"status\":{}}\n' ({\"metadata\":{\"name\":\"harvester ...)"
time="2024-08-01T06:43:51Z" level=error msg="error syncing 'ntp-servers': handler harvester-setting-controller: the object provided is unrecognized (must be of type NodeConfig): Object 'Kind' is missing in '{\"metadata\":{\"name\":\"harvester-node-0\",\"namespace\":\"harvester-system\",\"creationTimestamp\":null},\"spec\":{\"ntpConfigs\":{\"ntpServers\":\"0.suse.pool.ntp.org 1.suse.pool.ntp.org\"}},\"status\":{}}\n' ({\"metadata\":{\"name\":\"harvester ...), requeuing"
W0801 06:44:50.656326       7 warnings.go:70] kubevirt.io/v1 VirtualMachineInstancePresets is now deprecated and will be removed in v2.
time="2024-08-01T06:45:51Z" level=info msg="Processing setting ntp-servers, value: {\"ntpServers\":[\"0.suse.pool.ntp.org\",\"1.suse.pool.ntp.org\"]}"
time="2024-08-01T06:45:51Z" level=error msg="Create node config with nodeName harvester-node-0 failure: the object provided is unrecognized (must be of type NodeConfig): Object 'Kind' is missing in '{\"metadata\":{\"name\":\"harvester-node-0\",\"namespace\":\"harvester-system\",\"creationTimestamp\":null},\"spec\":{\"ntpConfigs\":{\"ntpServers\":\"0.suse.pool.ntp.org 1.suse.pool.ntp.org\"}},\"status\":{}}\n' ({\"metadata\":{\"name\":\"harvester ...)"
time="2024-08-01T06:45:51Z" level=error msg="error syncing 'ntp-servers': handler harvester-setting-controller: the object provided is unrecognized (must be of type NodeConfig): Object 'Kind' is missing in '{\"metadata\":{\"name\":\"harvester-node-0\",\"namespace\":\"harvester-system\",\"creationTimestamp\":null},\"spec\":{\"ntpConfigs\":{\"ntpServers\":\"0.suse.pool.ntp.org 1.suse.pool.ntp.org\"}},\"status\":{}}\n' ({\"metadata\":{\"name\":\"harvester ...), requeuing"
```

**To Reproduce**
Create harvester cluster from master branch and run `kubectl logs deployments/harvester -n harvester-system`.

**Expected behavior**
there should have no error log syncing ntp-server

**Support bundle**
N/A

**Environment**
 - Harvester ISO version:  master branch

**Additional context**
N/A

**Comment by Yu-Jack** [comment]
It also happens in v1.4-head and it's a wrangler version issue.

**Comment by brandboat** [comment]
We've discussed this issue offline, and the root cause is that `node-manager` use old wrangler package which isn't match the wrangle version in harvester repo.

**Comment by Vicente-Cheng** [comment]
We will bump wrangler v3 on node-manager and bump the latest node-manager on the harvester.
That we could use the latest generated function (from node-manager). 

Kudos to @Yu-Jack, @FrankYang0529, @brandboat !

**Comment by brandboat** [comment]
closed as this is resolved in v1.4 branch and master
